specs/ukhpi-data-display.spec

```diff
@@ -0,0 +1,118 @@
+# UKHPI data display
+
+Tags: UKHPI
+
+## Query results default layout
+
+Tags: data, results
+
+* Default query is set to 'United Kingdom' and 'October 2020 to October 2021'
```

Comment:
Actually the default will be to the latest month for which data is available, to 12 months prior to that. So it will change depending on when the most recent publication was. The "to:" date should, however, never be more than 2 months ago.